gradle.properties

```diff
@@ -11,4 +11,6 @@ developerId=cloudinary
 developerName=Cloudinary
 developerEmail=info@cloudinary.com
 publishGroupId=com.cloudinary
-publishVersion=0.0.1-alpha.1
\ No newline at end of file
+publishVersion=0.0.1-alpha.1
+android.useAndroidX=true
```

Comment:
related to this PR?

Reply:
It's part of the cleanup. Not critical for this PR, but it is a _cleanup_ change so I don't think there's a need to separate the commits.